src/class_objectProcessor.py

```diff
@@ -792,7 +776,7 @@ def processbroadcast(self, data):
             except:
                 apiNotifyPath = ''
             if apiNotifyPath != '':
-                call([apiNotifyPath, "newBroadcast"])
+                call([apiNotifyPath, "newBroadcast"])  # nosec
```

Comment:
Here also perhaps a "TODO: sanitise"

Reply:
I believe that this is already sanitised (only looks to settings on local disk with fixed parameter).